libnavui-maps/api/current.txt

```diff
@@ -878,18 +874,18 @@ package com.mapbox.navigation.ui.maps.route.line.model {
     method public com.mapbox.navigation.ui.maps.route.line.model.MapboxRouteLineOptions.Builder withVanishingRouteLineEnabled(boolean isEnabled);
   }
 
-  public final class RouteFeatureData {
```

Comment:
Removal of the `RouteFeatureData` from the public API is intentional here. We most-likely exposed it by mistake since there is no practical use for it.